REDMAX

jdigne opened the pull request:
new file:   data/siggraph-2019/10.1145-3306346.3322952.json

This Pull Request introduces changes to the following papers:
 * [ PUT ACM Ref heres ]

Thanks for contributing to this project. Contribution rules and help are available here: https://replicability.graphics/index.html#contribute.

Your contribution will be validated by project owners. Upon acceptance, it will appear automatically on the website https://replicability.graphics

**Note about self-reviewing**: we accept the reviews provided by the authors of a paper. These reviews will be clearly identified and may be double checked to validate the replicability. If you're an author, please clearly mention it when submitting your review (pull-request or email).

jdigne committed:
REDMAX
	new file:   data/siggraph-2019/10.1145-3306346.3322952.json

dcoeurjo commented on data/siggraph-2019/10.1145-3306346.3322952.json:
```diff
@@ -0,0 +1,96 @@
+        "git/hg/svn commit hash or revision number": "299d70d9713385661ce94ee5f2011470002b46bf",
+        "git/hg/svn commit hash or revision number URL2": "",
+
+        "Software Heritage permalink": "",
```
There is a SWH link for this one https://archive.softwareheritage.org/browse/origin/log/?origin_url=https://github.com/sueda/redmax

But I cannot cut/paste the Permalimk on my phone..

jdigne replied:
done

jdigne commented on data/siggraph-2019/10.1145-3306346.3322952.json:
```diff
@@ -0,0 +1,96 @@
+        "git/hg/svn commit hash or revision number": "299d70d9713385661ce94ee5f2011470002b46bf",
+        "git/hg/svn commit hash or revision number URL2": "",
+
+        "Software Heritage permalink": "",
```
```suggestion
        "Software Heritage permalink": "https://archive.softwareheritage.org/swh:1:rev:3251bcb93cd91a892a8a056b6536a295b0ed80be;origin=https://github.com/sueda/redmax;visit=swh:1:snp:7a235a6948ea44d6bc90942c935c9234831ee01e/",
```

jdigne committed:
Update data/siggraph-2019/10.1145-3306346.3322952.json

jdigne committed:
finished papers from "motion is in control" session
	new file:   data/siggraph-2019/10.1145-3306346.3322938.json
	new file:   data/siggraph-2019/10.1145-3306346.3323003.json
	new file:   data/siggraph-2019/10.1145-3306346.3323034.json

jdigne committed:
Merge branch 'motion_control' of github.com:GraphicsReplicability/replicability.graphics into motion_control
